treflun/urls.py

```diff
@@ -1,24 +1,28 @@
-"""treflun URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/3.1/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
-from django.contrib import admin
-from django.urls import path, include
-from django.conf.urls.static import static
-from treflun import settings
-
-urlpatterns = [
-    path('admin/', admin.site.urls),
-    path('', include('game.urls'))
-] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+"""treflun URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/3.1/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib.auth.views import LoginView
+from django.contrib import admin
+from django.urls import path, include
+from django.conf.urls.static import static
+
+from treflun import settings
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path('', LoginView.as_view(template_name='accounts/login.html')),
+    path('game', include('game.urls')),

```

Comment:
`'game/'` to be consisted with other patterns

Reply:
Done